tests/providers/cncf/kubernetes/operators/test_pod.py

```diff
@@ -179,7 +179,7 @@ def test_templates(self, create_task_instance_of_operator):
         assert dag_id == ti.task.arguments
         assert dag_id == ti.task.env_vars[0]
         assert dag_id == rendered.annotations["dag-id"]
-        assert dag_id == ti.task.configmaps[0]
+        assert dag_id == ti.task.env_from[0].config_map_ref.name
```

Comment:
Can you add the test cases for other env sources as well?

Reply:
Which other sources? What is your concern? The change only applies to configmaps

Reply:
Yep. Should be fine.